ldcbench.http-node/src/main/java/org/dice_research/ldcbench/nodes/http/simple/dump/DumpFileResource.java

```diff
@@ -33,19 +36,25 @@
     public static final List<Archiver> ARCHIVERS = Arrays.asList(new TarArchiver(),
             new TarArchiver(ReflectionBasedStreamFactory.create("java.util.zip.GZIPOutputStream", "application/gzip", ".gz")),
             new ZipArchiver());
+    public static final ArrayList<File> multipleFile = new ArrayList<File>();
```

Comment:
Why do you need a public static attribute? This should not be the case! 
It is a sign that something is not correct with the class structure! Please remove this attribute and find a better solution.